### ISSUE TITLE
README examples not working

### ISSUE THREAD

**Issue by Midnoclose** [slages/love-imgui#37]
Whenever I try to run either of the examples in the README, I get an error similar to this:
```
Error

main.lua:18: bad argument #1 to 'NewFrame' (string expected, got table)


Traceback

[C]: in function 'NewFrame'
main.lua:18: in function 'update'
[C]: in function 'xpcall'
```

**Comment by flamendless** [comment]
Yes, this happens with the 0.9 release version. I am also looking at this issue

**Comment by Moonstrous** [comment]
Any luck ?

**Comment by Hperigo** [comment]
Hi, just in case somebody needs help:

I'm on MacOs and using the released bin from github gave me the same error. But building the library worked

**Comment by nullset2** [comment]
I am trying to cross-reference patrickriordan's imgui_lua_bindings, which I believe this is based off of, and I'm trying to use the Drawlist from lua to no avail:

```
    imgui.DrawList_AddLine(
        imgui.DrawList_AddLine(0, 0, 100, 100, 0xFF0000FF, 2));
```

I'm trying to come up with an example as to how to use the drawlist but nothing has come up. An example in this repo would also be nice.

**Comment by nexovec** [comment]
Is this a project no longer maintained? I'm getting the same issue.

**Comment by flamendless** [comment]
Use the mikuauah fork for updated one 

https://github.com/MikuAuahDark/love-imgui
